app/models/game.rb

```diff
@@ -3,4 +3,124 @@ class Game < ApplicationRecord
   has_many :pieces
 
   enum game_status: { in_progress: 0, checkmate: 1, stalemate: 2 }
+
+  def populate_board!
+    # this should create all 32 pieces with their initial X/Y coordinates.
+    
+    # Build white pieces
+
+    (0..7).each do |i|
+      Pawn.create(
+        row_coordinate: 6,
+        column_coordinate: i,
+        color: "white",
+        )
```

Comment:
One thing missing from each of these `creates` is the `game_id`.

What you can do is...

`pieces.create(type: 'Pawn', #etc...)`

By calling it on `pieces`, you're telling Rails this new piece belongs to `game`, because `pieces` is a `has_many` association on this model already.